### ISSUE TITLE
When creating two different groups with the same name - crashes

### ISSUE THREAD

**Issue by derto42** [derto42/FlowBuddy#23]
```Traceback (most recent call last):
  File "D:\System\Pictures\AI\Projects\DesignOverlay\GitHub\src\main.py", line 551, in create_new_group
    SF.new_group(group_name)
  File "D:\System\Pictures\AI\Projects\DesignOverlay\GitHub\src\SaveFile.py", line 221, in new_group
    raise ValueError("Group name already exists.")
ValueError: Group name already exists.```

**Comment by VasigaranAndAngel** [comment]
this bug waiting to be fixed when the order is implemented.

**Comment by VasigaranAndAngel** [comment]
#39 will fix this by popping up a warning message if the group/task already exists.

**Comment by VasigaranAndAngel** [comment]
Fixed in eb0d3226d923b02e7b0543a9e4449b747b026a2f commit
